src/mount/chunk_writer.cc

```diff
@@ -436,6 +436,48 @@ void ChunkWriter::fillOperation(Operation &operation, int first_block, int first
 	}
 }
 
+/*!
+ * Fills given operation with a range of blocks beyond file size required to be read.
+ * \param operation operation to be filled
+ * \param first_block first block of the operation
+ * \param first_index index of the first block required to be read
+ * \param size number of required blocks
+ * \param stripe_element map of blocks in a stripe
+ */
+void ChunkWriter::fillNotExisting(Operation &operation, int first_block,
```

Comment:
Consider declaring first_block parameter as unsigned integer.

Reply:
We will leave it for now to match with already existing code.
![image](https://github.com/leil-io/saunafs/assets/22012909/5beb2dc8-71bd-4cb8-8aeb-80964abb1980)